src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -72,14 +69,8 @@ const CategoryDropdown = ({
     })
   }
 
-  const createCategory = useCallback(
-    (params?: CreateCategoriesParams) =>
-      ResourceService.createResourceCategory(params),
-    []
-  )
-
   const onResponseCategory = useCallback(
-    async (response: Categories | null) => {
+    (response: Categories | null) => {
```

Comment:
In `ResourceService.createResourceCategory` you return only `Category`, why here `null` is present?
```suggestion
    (response: Categories) => {
```

Reply:
Thank you, didn`t notice